python3.8/cookiecutter-aws-sam-hello-http-python/{{cookiecutter.project_name}}/template.yaml

```diff
@@ -0,0 +1,44 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  {{ cookiecutter.project_name }}
+
+  Sample SAM Template for {{ cookiecutter.project_name }}
+
+# More info about Globals: https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst
+Globals:
+  Function:
+    Timeout: 3
+
+Resources:
+  HelloWorldFunction:
+    Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+    Properties:
+      CodeUri: hello_world_http/
+      Handler: app.lambda_handler
+      {%- if cookiecutter.runtime == 'python3.7' %}
+      Runtime: python3.7
+      {%- elif cookiecutter.runtime == 'python3.8' %}
+      Runtime: python3.8
+      {%- elif cookiecutter.runtime == 'python3.9' %}
+      Runtime: python3.9
```

Comment:
This example is under the "python3.8" folder so we shouldn't be creating a python3.9 function. @mndeveci Do you want this to be a python3.9 example instead? I can move it or we can leave it as python3.8 for now.

Reply:
I have updated my fork to bring the example into the python3.9 directory, and as suggested I have made python3.9 the default runtime.
Tested with the following commands:
(starting in the example cookiecutter directory)
sam init --location .
cd aws-sam-hello-http-python
sam build --use-container
sam local invoke
sam deploy --guided

NB 'sam deploy --guided' succeeded only if I selected the default region us-east-1.  
If I selected ap-southeast-2 it failed with an error related to the autogenerated S3 bucket containing required artifacts.

Reply:
1. The warning about the reserved logical ID is a known bug in SAM CLI that we haven't addressed yet.
2. I'll try to check if I have the same issue.
Otherwise, the changes look good to me. Thanks for your contribution @tim-littlefair !

Reply:
I could deploy successfully in ap-southeast-2 by running `sam deploy --guided`. I believe the issue you're facing is something unrelated